packages/client/src/runtime/core/init/validateEngineInstanceConfig.test.ts

```diff
@@ -91,8 +66,6 @@ describe('validateEngineInstanceConfig', () => {
     test('do not recommend using `--no-engine` for ClientEngine', () => {
```

Comment:
this test should be removed completely i think

Reply:
Why? `--no-engine` no longer exists in the codebase